### PR TITLE
iOS release notes for version 1.143.0

### PR DESCRIPTION
**Pull request by msamosij**

### CODE REVIEW

File: _docs/ios/1.143.0.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: 1.143.0
+permalink: /docs/ios/1.143.0/
+parent: iOS
+nav_order: -143
+---
+
+## The logs from devices can be collected via Bluetooth and shared with our support
+Device logs page had a new feature that allows to collect logs from Lock, Bridge or Keypad via Bluetooth. In case of some issue, that logs will help us in quicker identify root cause of the issue and to make our products better in future.\
```
Comment (alzakharenk) [inline _docs/ios/1.143.0.md:10]: unused \

Comment (msamosij) [reply _docs/ios/1.143.0.md:10]: This is line brake character. Without it, sometimes text was wrap around the image.

Comment (matelig) [inline _docs/ios/1.143.0.md:10]: unnecessary '\\'

**Review by matelig — APPROVED** [review]

File: _docs/ios/1.143.0.md (hunk 1 above)
Comment (msamosij) [reply _docs/ios/1.143.0.md:10]: This is line brake character. Without it, sometimes text was wrap around the image.

**Pull request merged**